src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -38,6 +38,7 @@ sealed trait Task {
     taskId.mesosTaskId
   }
 
+  // FIXME (merge): remove this accessor and add a universal Status to all tasks that includes unconfirmed states
```

Comment:
I've removed the fixme for now.
(I'll see if I can safely add the mentioned functionality on top.)